geode-redis/src/main/java/org/apache/geode/redis/internal/executor/ExpireExecutor.java

```diff
@@ -26,8 +28,6 @@
 
 public class ExpireExecutor extends AbstractExecutor implements Extendable {
 
-  private final String ERROR_SECONDS_NOT_USABLE = "The number of seconds specified must be numeric";
-
   private final int SECONDS_INDEX = 2;
```

Comment:
`private finals` could be local.  Though maybe since these are the same as ExpireAtExecutor they should be abstracted?  Though I would also wait for rule of three duplications...

Reply:
Localified.